packages/components/psammead-caption/src/index.jsx

```diff
@@ -1,17 +1,24 @@
-import styled from 'styled-components';
+import styled, { css } from 'styled-components';
 import { GEL_SPACING, GEL_SPACING_DBL } from '@bbc/gel-foundations/spacings';
 import { MEDIA_QUERY_TYPOGRAPHY } from '@bbc/gel-foundations/breakpoints';
 import {
   GEL_LONG_PRIMER,
   GEL_FF_REITH_SANS,
 } from '@bbc/gel-foundations/typography';
-import { C_STONE, C_STORM } from '@bbc/psammead-styles/colours';
+
+import { C_CLOUD_DARK } from '@bbc/psammead-styles/colours';
+
+// Defined separately since in future will need to apply
+// only when the script supports italic text
+const FS_ITALIC = css`
```

Comment:
extracting this because it will be used as a feature seems a bit feature creepy to me, but not overly fussed 🙈

Reply:
Fair point. I thought it'd help the WS team's work on localising the component. Likely that'd be better done via discussion rather than code comments. :)